examples/browserify/gulpfile.js

```diff
@@ -21,12 +21,12 @@ gulp.task('build-bundle', function() {
 gulp.task('build-worker', function() {
   // We can create our own viewer (see worker.js) or use already defined one.
   var workerSrc = require.resolve('pdfjs-dist/build/pdf.worker.entry');
-  return browserify(workerSrc, {output: TMP_FILE_PREFIX + 'worker.tmp'})
+  return browserify(workerSrc, { output: TMP_FILE_PREFIX + 'worker.tmp', })
     .bundle()
     .pipe(source(TMP_FILE_PREFIX + 'worker.tmp'))
-    .pipe(streamify(uglify({compress:{
-      sequences: false // Chrome has issue with the generated code if true
-    }})))
+    .pipe(streamify(uglify({ compress: {
+      sequences: false, // Chrome has issue with the generated code if true
+    }, })))
```

Comment:
This isn't very readable, even as-is in `master`, let's reformat along the lines of:
```javascript
.pipe(streamify(uglify({
  compress: {
    sequences: false, // Chrome has issue with the generated code if true
  },
})))
```